### PR TITLE
Format all models for consistency

### PR DESCRIPTION
**Pull request by poloaraujo**
## Overview

Currently there are inconsistencies in the SQL scripts styles used (e.g. trailing vs leading commas). I've formatted al sources, staging models, fact models and core models (both dimensional and fact) to have a consistent format for both spacing and commas (of which all use trailing)

## Update type - breaking / non-breaking

- [ ] Minor bug fix
- [ ] Documentation improvements
- [ ] Quality of Life improvements
- [ ] New features (non-breaking change)
- [ ] New features (breaking change)
- [x] Other (non-breaking change)
- [ ] Other (breaking change)
- [ ] Release preparation

## What does this solve?

Issue #383 

## Outstanding questions

So far, I've only tested locally with Snowflake

## What databases have you tested with?

- [x] Snowflake
- [ ] Google BigQuery
- [ ] Databricks
- [ ] Spark
- [ ] N/A

### CODE REVIEW

**Review by glsdown — APPROVED** [review]
Pending approval from our robot overlords (i.e. the CI Linter) this looks great to me! Thank you for doing this. 🎉

File: models/dim_dbt__current_models.sql [hunk 1]
```diff
@@ -1,97 +1,113 @@
...
-        {% endif %}
-    from latest_models
-    left join latest_model_stats
-        on latest_models.node_id = latest_model_stats.node_id
-)
+with
+    base as (
+
+        select
+            *
```
Comment (llifoawing) [inline models/dim_dbt__current_models.sql]: SQLFluff raised LT09 - single select target doesnt need to be on new line

File: models/dim_dbt__current_models.sql [hunk 2]
```diff
@@ -1,97 +1,113 @@
...
+        from {{ ref('stg_dbt__models') }}
+
+    )
+
+    , model_executions as (
+
+        select
+            *
```
Comment (llifoawing) [inline models/dim_dbt__current_models.sql]: SQLFluff raised LT09 - single select target doesnt need to be on new line

File: models/dim_dbt__current_models.sql [hunk 3]
```diff
@@ -1,97 +1,113 @@
...
+        from {{ ref('stg_dbt__model_executions') }}
+
+    )
+
+    , latest_models as (
+
+        /* Retrieves the models present in the most recent run */
+        select
+            *
```
Comment (llifoawing) [inline models/dim_dbt__current_models.sql]: SQLFluff raised LT09 - single select target doesnt need to be on new line

**Review by llifoawing — CHANGES_REQUESTED** [review]
Hey @poloaraujo - resurrecting this PR so we can merge your changes!
- SQLfluff failed due to 3 single select targets being on new lines.
- I was running into errors when running tox locally related to this [issue](https://github.com/sqlfluff/sqlfluff/issues/5841#issuecomment-2135297644) - can you upgrade sqlfluff-templater-dbt~=3.0.7

**Review by llifoawing — APPROVED** [review]

**Pull request merged**